src/commands/subdomain.rs

```diff
@@ -82,44 +65,24 @@ fn register_subdomain(
     message::working(&msg);
     let account_id = &target.account_id;
     let addr = subdomain_addr(account_id);
-    let sd = Subdomain {
+    let subdomain = Subdomain {
         subdomain: name.to_string(),
     };
-    let sd_request = serde_json::to_string(&sd)?;
+    let subdomain_request = serde_json::to_string(&subdomain)?;
 
     let client = http::auth_client(None, user);
 
-    let mut res = client.put(&addr).body(sd_request).send()?;
+    let mut res = client.put(&addr).body(subdomain_request).send()?;
```

Comment:
This call to subdomain API looks redundant--it seems like we make the exact same call later in Subdomain::get(). Is there any way we can consolidate this into just one call?

Reply:
There doesn't really seem to be a way around this - we already make two requests to the API in the existing implementation. In fact I think we make three and this PR reduces it to two. The `put` request doesn't seem to give us info about _why_ the put failed, so we must perform a `get` as well to determine if they already have a subdomain or not.

Reply:
Ahh! I see. I misread and thought both were GETs. This makes sense now.